CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # [develop](https://github.com/adhearsion/electric_slide)
+  * Added `ElectricSlide::CallQueue#update_agent` to safely update a queued agent object's attributes
```

Comment:
The original intent was simply to use `add_agent` to either add or update as appropriate. This is safer as it reduces race conditions from outside callers (is the agent already in the queue or not?) Can you explain what the motivation is here for having a separate method?